src/events.types.ts

```diff
@@ -0,0 +1,42 @@
+export enum EventType {
+    CREATE,
+    BUY,
+    SELL
+}
+
+export interface CreateEvent {
+    type: EventType.CREATE,
+    name: string,
+    id: string,
+}
+
+export interface BuyEvent {
+    type: EventType.BUY,
+    id: string,
+    symbol: string,
+    quantity: number,
+    date: Date,
+    price: number,
+}
+
+export interface SellEvent {
+    type: EventType.SELL,
+    id: string,
+    symbol: string,
+    quantity: number,
+    date: Date,
+    price: number,
+}
+
+export const TRADER_EVENTS: Map<[trader_id: string], Event[]> = new Map()
```

Comment:
Would probably locate this elsewhere in our app design. `infra/db/eventstream.ts`, for example

Overall, I like this approach and execution in terms of the event types and how they're stored in this stream